charmander/models/order_event_type.py

```diff
@@ -74,6 +79,7 @@ class OrderEventType(EventType):
         ),
     )
 
+    # automatic after submitting details FOR NOW XXX
```

Comment:
This is a comment for the toy transition I added in the factory above. Eventually we might actually listen for an event from an external service and the code in the factory is just a placeholder until that is ready.